lib/ui/settings/interactor/settings_bloc.freezed.dart

```diff
@@ -1417,7 +1417,7 @@ class __$$_SettingsStateCopyWithImpl<$Res>
 
 class _$_SettingsState implements _SettingsState {
   const _$_SettingsState(
```

Comment:
part of this PR?

Reply:
No, This was generated when i called `flutter pub run build_runner` looks like we committed code to main without running the `flutter pub run build_runner`